src/components/Pagination/types.ts

```diff
@@ -0,0 +1,11 @@
+export type TPaginationProps = {
+  nextLabel?: string | JSX.Element
+  prevLabel?: string | JSX.Element
+  totalCount: number
+  page: number
+  handleChangePage: (pageNumber: number) => void
+  breakLabel?: JSX.Element
```

Comment:
can breakLabel be something else besides `more icon`?

Reply:
according to the designer, only this option can be used